packages/slice-machine/pages/slices.tsx

```diff
@@ -179,39 +169,32 @@ const SlicesIndex: React.FunctionComponent = () => {
                         </Flex>
                         <Grid
                           elems={components}
-                          defineElementKey={(slice: ComponentUI) =>
-                            slice.model.name
-                          }
-                          renderElem={(slice: ComponentUI) => {
-                            return SharedSlice.render({
-                              slice,
-                              StatusOrCustom: {
-                                status: modelsStatuses.slices[slice.model.id],
-                                authStatus,
-                                isOnline,
-                              },
-                              actions: {
-                                onUpdateScreenshot: (e: React.MouseEvent) => {
-                                  e.preventDefault();
-                                  onOpenModal({
-                                    sliceFilterFn: (s: ComponentUI[]) =>
-                                      s.filter(
-                                        (e) => e.model.id === slice.model.id
-                                      ),
-                                  });
-                                },
-                                openRenameModal: (slice: ComponentUI) => {
+                          defineElementKey={(slice) => slice.model.name}
+                          renderElem={(slice) => (
+                            <SharedSliceViewCard
+                              action={{
```

Comment:
❓ Kind of same question as one before. Why not exporting different kind of component here for the action?